core/src/main/java/io/kestra/core/runners/RunContextLogger.java

```diff
@@ -294,7 +295,13 @@ protected void append(ILoggingEvent e) {
             e = this.transform(e);
 
             logEntries(e, logEntry)
-                .forEach(logQueue::emitAsync);
+                .forEach(log -> {
+                    try {
+                        logQueue.emitAsync(log);
+                    } catch (QueueException ex) {
+                        // silently do nothing
```

Comment:
I think a catch should never do nothing, at least log something

Reply:
In 99% of cases you"re right, but here is the exception to this rule, if we cannot log (which will never happen), logging in the server log is not necessary as the log would already have been emitted to it (and moreover, in this case something really wrong is happening so there should already be thousands of logs in the server log.

If it's very important to you I can add a debug log thought ...